drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -299,13 +300,22 @@ function DragAndDropTemplates(configuration) {
             attemptsUsedInfo = h("span.attempts-used", {id: attemptsUsedId}, attemptsUsedText);
         }
 
+        var submitSpinner = null;
+        if (ctx.submit_spinner) {
+            submitSpinner = h('span', [
+                h('span.fa.fa-spin.fa-spinner', {attributes: {'aria-hidden': true}}),
+                h('span.sr', gettext('Saving'))
+            ]);
+        }
+
         return (
             h("div.action-toolbar-item.submit-answer", {}, [
                 h(
                     "button.btn-brand.submit-answer-button",
                     submitButtonProperties,
                     [
-                        (ctx.submit_spinner ? h("span.fa.fa-spin.fa-spinner") : null),
+                        submitSpinner,
+                        ' ',  // whitespace between spinner icon and text
                         gettext("Submit")
```

Comment:
I'm not sure any sr text is necessary here since the button is disabled and the text "Submit" is present. Generally one only needs screen-reader text when there is no displayed text.

I'd suggest you just run this case by @cptvitamin and see what he says.